thieme-german.csl

```diff
@@ -24,22 +24,22 @@
     <category citation-format="numeric"/>
     <category field="medicine"/>
     <summary>Modified version of the "Vancouver Bracket" for Thieme Journals</summary>
-    <updated>2017-05-18T15:23:08+00:00</updated>
+    <updated>2020-06-30T08:44:34+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <macro name="author">
-    <names variable="author" suffix=". ">
-      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always"/>
-      <label form="short" prefix=" (" suffix=")"/>
+    <names suffix=". " variable="author">
+      <name sort-separator=" " name-as-sort-order="all" initialize-with="" et-al-use-first="3" et-al-min="4" delimiter-precedes-last="always"/>
+      <label prefix=" (" form="short" suffix=")"/>
       <substitute>
-        <names variable="editor"/>
+        <names variable="author"/>
```

Comment:
Substituting "author" for "author" here doesn't make sense. Maybe just remove it.